metecho/api/jobs.py

```diff
@@ -207,11 +204,25 @@ def _create_org_and_run_flow(
     scratch_org.owner_sf_username = sf_username or user.sf_username
     scratch_org.owner_gh_username = user.username
     scratch_org.save()
+
+    org_config = cci.project_config.keychain.get_org(scratch_org.task.org_config_name)
```

Comment:
@wlonk Oops. This is overriding the `org_config` set above from `create_org`.

@davisagli Related -- that `create_org` is hardcoding `org_name = "dev"` -- should we be instead passing in `scratch_org.task.org_config_name` to the creation as well?

Reply:
Yes, create_org needs to look up the org config that was selected by the user so that the org gets created using the correct settings. _create_org_and_run_flow can then make use of the scratch_org_config returned from create_org to check for setup_flow rather than re-fetching the org config from the keychain